### PR TITLE
Replace streamConfig with streamConfigMaps

### PR DESCRIPTION
**Pull request by noramullen1**
Addresses https://cortexdata.atlassian.net/browse/DOCS-65

Multiple updates to [Table config doc](https://docs.pinot.apache.org/configuration-reference/table) and [Streaming ingestion](https://docs.pinot.apache.org/basics/data-import/pinot-stream-ingestion#create-ingestion-configuration) doc
- Update examples on both to use streamConfigMaps
- Replace streamConfig table in Streaming ingestion doc with streamConfigMaps table
- Remove now-obsolete streamConfig table from current version of Table config doc and link to streamConfigMaps
- Remove redundant/irrelevant table property list on Stream ingestion doc

### CODE REVIEW

File: basics/data-import/pinot-stream-ingestion/README.md [hunk 1]
```diff
@@ -77,40 +78,31 @@ For our sample data, the schema configuration looks like this:
...
+| `stream.[streamType].decoder.prop.format`             |                                                                                                                                                                                       | - `JSON`                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (noramullen1) [inline basics/data-import/pinot-stream-ingestion/README.md]: Need a definition for this one

File: configuration-reference/table.md [hunk 2]
```diff
@@ -394,14 +329,14 @@ Here's an example table config for a realtime table. **All the fields from the o
         "stream.kafka.zk.broker.url": "XXXX",
         "streamType": "kafka"
       }
+    ]
     },
-    "tenants": {
+    "tenants":{[
```
Comment (kelseiv) [inline configuration-reference/table.md]: to be sure, did we want `[` here?

File: basics/data-import/pinot-stream-ingestion/README.md (hunk 1 above)
Comment (kelseiv) [reply basics/data-import/pinot-stream-ingestion/README.md]: for now, we could say something like:
Specifies the data format to ingest via a stream. The value of this property should match the format of the data in the stream.

**Review by kelseiv — APPROVED** [review]

File: configuration-reference/table.md (hunk 2 above)
Comment (noramullen1) [reply configuration-reference/table.md]: no! good catch

**Review by lewy-startree — COMMENTED** [review]
LGTM

**Pull request merged**